packages/wdio-utils/src/utils.ts

```diff
@@ -162,7 +162,7 @@ export function safeRequire (name) {
          */
         const localNodeModules = path.join(process.cwd(), '/node_modules')
         /* istanbul ignore if */
-        if (!require.main.paths.includes(localNodeModules)) {
+        if (require && require.main && !require.main.paths.includes(localNodeModules)) {
```

Comment:
The following should work as well.

```suggestion
        if (!require?.main?.paths.includes(localNodeModules)) {
```